charts/incubator/fancyindex/values.yaml

```diff
@@ -0,0 +1,13 @@
+image:
+  repository: xal3xhx/fancy
+  pullPolicy: IfNotPresent
+  tag: latest
+
+env: {}
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 80
+        port: 10081
```

Comment:
port needs to be a free port from truecharts/pub/manual/default-ports.md
(probably something like 102xx). 
Also needs to be a PR there to add it to the list